src/ABC.cpp

```diff
@@ -45,6 +45,17 @@ struct ABC : Module {
 		configParam(C1_LEVEL_PARAM, -1.0, 1.0, 0.0, "C1 Level");
 		configParam(B2_LEVEL_PARAM, -1.0, 1.0, 0.0, "B2 Level");
 		configParam(C2_LEVEL_PARAM, -1.0, 1.0, 0.0, "C2 Level");
+
+		configInput(A1_INPUT, "A1");
+		configInput(B1_INPUT, "B1 (normalled to 5V)");
```

Comment:
Remove the description from ports and add it to
```cpp
paramQuantities[XYZ_INPUT]->description = "Normalled to 5V";
```

Reply:
I guess in this case the inputs have a corresponding param (knob), but that's not always the case. Unless I'm missing something, it doesn't look like inputs can have an independent description from their name. It's a shame to lose this information (as I often find myself wondering what CV range a module will accept), but happy to go with your preference.

If you meant update the corresponding knob to say, e.g., "normalled to 10V", I don't think that makes any more sense.

Reply:
My apologies, my code should be
```cpp
getInputInfo(B1_INPUT)->description = "Normalled to 5V";
```
Port tooltips should show descriptions too. Let me know if it doesn't work.